### PR TITLE
Adding new Koji temperature post

### PR DESCRIPTION
**Pull request by CamDavidsonPilon**
@daizu27, overall, great article! The article made clear the role of evaporation and moisture-content in koji making, and importantly, connected it to how homemade koji can be engineered better :+1:

1. I made some additions to **Water, the Weight Ratio, and the Glucose Effect** - I believe that effect of increasing water on the glucose effect is caused by (↑water → ↑enzyme activity → glucose inhibition) - but let me know if you think otherwise. (The original reasoning you wrote, if I understood correctly, was due to increasing uptake of glucose, which is probably true, but would be less of an effect than enzyme activity increasing) 

2. I want to propose a new introduction paragraph. Feel free to write one up. On the other hand, I've wrote something below, as an example or something we can go with. 

Koji is a living organism, and like all organisms, it generates a considerable amount of heat. If you've ever made koji at home, it's almost unsettling to feel the warmth of the koji cake. Controlling that heat, with water, mixing, and air circulation, is crucial to ending up with a quality product that's high in enzymes. In this article, we'll dive into how temperature is both controlled and *exploited* to create a better final product.

### CODE REVIEW

File: _posts/2020-12-16-the-koji-making-process-temperature-mycelium-and-moisture.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+### Mycelia and "Haze"
+
+Japanese brewers have a term called *"haze"* (pronounced like *saké*) that describes the state of mycelial growth growing on a grain of koji. The term is further distinguished into two concepts:
+
+1. *Haze-mawari* - how to koji mycelium spreads on a grain's **surface**
+2. *Haze-komi* - how the koji mycelium penetrates into the grain's **interior**
+
+Koji with a high enzyme concentration must have good growth on the surface of the grain, the centre of the grain, and have the proper moisture content. Brewers have different terms to describe the final product koji's visible surface and interior growth:
+
+{:.table-clean}
```
Comment (CamDavidsonPilon) [inline _posts/2020-12-16-the-koji-making-process-temperature-mycelium-and-moisture.md:60]: using this class, the tables appear more "table-like":

Screen Shot 2020-12-16 at 10 38 30 AM

Comment (daizu27) [reply _posts/2020-12-16-the-koji-making-process-temperature-mycelium-and-moisture.md:60]: Noted - thanks. Looks good.

**Comment by daizu27** [comment]
> @daizu27, overall, great article! The article made clear the role of evaporation and moisture-content in koji making, and importantly, connected it to how homemade koji can be engineered better 👍
> 
> 1. I made some additions to **Water, the Weight Ratio, and the Glucose Effect** - I believe that effect of increasing water on the glucose effect is caused by (↑water → ↑enzyme activity → glucose inhibition) - but let me know if you think otherwise. (The original reasoning you wrote, if I understood correctly, was due to increasing uptake of glucose, which is probably true, but would be less of an effect than enzyme activity increasing)
> 2. I want to propose a new introduction paragraph. Feel free to write one up. On the other hand, I've wrote something below, as an example or something we can go with.
> 
> Koji is a living organism, and like all organisms, it generates a considerable amount of heat. If you've ever made koji at home, it's almost unsettling to feel the warmth of the koji cake. Controlling that heat, with water, mixing, and air circulation, is crucial to ending up with a quality product that's high in enzymes. In this article, we'll dive into how temperature is both controlled and _exploited_ to create a better final product.

I read the modified paragraphs re: glucose effect. Looks good and clear to me. Agreed that the entire mechanism is driven by increased enzyme activity with increased water content.

Re: intro, I've adjusted the wording a bit. Let me know if this works. 

Koji generates a considerable amount of heat as it grows - if you've ever made koji at home, it's almost unsettling to feel the warmth of a koji cake. Controlling that heat, along with water content, mixing, and ventilation, is crucial to produce a quality product. In this article, we'll dive into how temperature and other factors are both controlled and _exploited_ in the koji-making process.

**Comment by CamDavidsonPilon** [comment]
lgtm! Merging on your final 👍

**Pull request merged**